File: medplum-java/medplum-model/src/main/java/com/fhir/MeasureReport_Stratum.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableMeasureReport_Stratum.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface MeasureReport_Stratum {
  Optional<List<Extension>> extension();

  Optional<List<Extension>> modifierExtension();

  Optional<String> id();

  Optional<List<MeasureReport_Population1>> population();

  Optional<CodeableConcept> value();

  Optional<List<MeasureReport_Component>> component();

  Optional<Quantity> measureScore();

  static ImmutableMeasureReport_Stratum.Builder builder() {
    return ImmutableMeasureReport_Stratum.builder();
  }
}

File: medplum-typescript/src/model/fhir/MeasureReport_Stratum.ts
import { CodeableConcept } from "./CodeableConcept";
import { Extension } from "./Extension";
import { MeasureReport_Component } from "./MeasureReport_Component";
import { MeasureReport_Population1 } from "./MeasureReport_Population1";
import { Quantity } from "./Quantity";

export interface MeasureReport_Stratum {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    value: CodeableConcept | null | undefined;
    component: MeasureReport_Component[] | null | undefined;
    population: MeasureReport_Population1[] | null | undefined;
    measureScore: Quantity | null | undefined;
}
